app/src/main/java/com/jnj/vaccinetracker/visit/screens/ReferralFragment.kt

```diff
@@ -124,7 +124,7 @@ class ReferralFragment : BaseFragment() {
     }
 
     private fun setupReferralWithinFacilityDropdown() {
-        val dropdownOptions = listOf("Value1", "Value2", "Value3")
+        val dropdownOptions = listOf("Nutrition Clinic", "EID (Early Infant Diagnosis) Clinic", "Post Natal Care Clinic","Family Planning Clinic", "Others Specify")
```

Comment:
Please remove it. I pushed yesterday new version of code that you didn't pull. Please add these clinic in separate PR (with the latest code)